src/goto-programs/validate_goto_model.cpp

```diff
@@ -48,7 +48,7 @@ class validate_goto_modelt
   /// - no return statements in any of the functions
   /// - lhs of every \ref code_function_callt instruction is nil
   /// - all return types are void (of both calls and functions themselves)
-  void check_returns_removed();
+  // void check_returns_removed();
```

Comment:
I think we should either completely remove this check, or else introduce a bit map for goto programs that indicates which passes have run (and then make this check conditional).

Reply:
Ok, will remove.

Reply:
done